libs/deposits/src/lib/deposit-manager.tsx

```diff
@@ -39,6 +41,12 @@ export const DepositManager = ({
     [assets, update]
   );
 
+  useEffect(() => {
+    if (assetId) {
+      handleSelectAsset(assetId);
```

Comment:
chore (blocking): Better to set the initial state than use an effect to update it

Reply:
updated and add an initial state on the asset. (same for above)